### DISCUSSION TITLE
Provide a way for orgs and repos to set a policy against running schedule events

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12180] [Actions] [answered]
I fork lots of repositories. Some will include workflows with `on: schedule`.

If a workflow *only* includes `on: schedule`, I can disable the workflow individually.

But, if a workflow has:
```workflow
on:
  push:
  pull_request:
  schedule: ...
```

And the push/pull_request events are used for standard CI, then I can't just disable the workflow if I want to know if I haven't broken the build.

I'd like to set a default policy on my personal account and one of my orgs to not run schedules. I'm not in a position to force each org/repo that I fork to not use scheduled tasks (I initially used them for my workflow, but have long since replaced that, so I do understand that repos may have had needs for the feature).

Note: I'm asking for policy options at the:
- org
- repo
- workflow
... level.

Setting at the org level should let me either ban them for my org, or default them to off.
Setting at the repo level should let me either ban them for my repo, or default them to off.
At the workflow level should let me enable or disable a schedule unless it's banned at the org or repo levels -- in which case, the `schedule` event souldn't fire).

This is *not* the same as disabling the workflow, as a `push` or `pull_request` or similar should still fire for the workflow, unless it's otherwise disabled.

**Comment by ethomson** [comment, marked as answer]
Thanks @jsoref - this is good feedback as usual.  This makes a lot of sense, we'll put this on the backlog.

**Comment by mkurz** [comment]
Related to this suggestion here, I think it totally would make sense to allow an author of a workflow to restrict for which repo a workflow is allowed to run (or if it is allowed to run it for forks) so I opened this ticket here: #16109
